pkg/controllers/dataexport/reconcile.go

```
@@ -812,6 +812,23 @@ func (c *Controller) stageSnapshotRestoreInProgress(ctx context.Context, dataExp
 		status: kdmpapi.DataExportStatusSuccessful,
 		reason: restoreInfo.Reason,
 	}
+	// Add annotation for corresponding PV to be skipped backing
+	tempName := toSnapshotPVCName(srcPvc.Name, string(dataExport.UID))
+	interPvc, err := core.Instance().GetPersistentVolumeClaim(tempName, srcPvc.Namespace)
+	if err != nil {
```

is the idea is to ignore the error?